frontend-student/src/components/coursePage/VideoSection.jsx

```
@@ -0,0 +1,77 @@
+    const match = url.match(regExp);
+
+    return match && match[2].length === 11
+      ? `https://www.youtube.com/embed/${match[2]}`
+      : null;
+  };
+
+  return (
+    <div>
+      {/* YouTube Video Player */}
+      <div className='w-full aspect-video bg-black rounded-lg overflow-hidden'>
+        {videoData && videoData.url && getYouTubeEmbedUrl(videoData.url) ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)